apps/astarte_appengine_api/lib/astarte_appengine_api/config.ex

```diff
@@ -128,6 +128,12 @@ defmodule Astarte.AppEngine.API.Config do
     type: :module,
     default: Astarte.RPC.AMQP.Client
 
+  @envdoc "The astarte instance id to generate keyspaces with"
+  app_env :astarte_instance_id, :astarte_appengine_api, :astarte_instance_id,
+    os_env: "ASTARTE_INSTANCE_ID",
+    default: "default",
```

Comment:
The default `astarte_instance_id` should be an empty string, otherwise this would be a breaking change.